bucket/mousejiggler.json

```diff
@@ -1,19 +1,20 @@
 {
-    "version": "1.8.42",
+    "version": "2.0.7",
     "description": "A very simple piece of software whose sole function is to \"fake\" mouse input to Windows, and jiggle the mouse pointer back and forth.",
-    "homepage": "https://github.com/cerebrate/mousejiggler",
+    "homepage": "https://github.com/cerebrate/mousejiggler/",
     "license": "MS-PL",
-    "url": "https://github.com/cerebrate/mousejiggler/releases/download/1.8.42/mousejiggler-1.8.42-net.zip",
-    "hash": "368df214d40bc26d8961ccdefcd43dd9303471c4338cfca13681ff0cbd1fd526",
-    "bin": "MouseJiggle.exe",
+    "url": "https://github.com/cerebrate/mousejiggler/releases/download/2.0.7/MouseJiggler.zip",
```

Comment:
It is 64bit only

![image](https://user-images.githubusercontent.com/13260377/105571815-6ef2b280-5d53-11eb-9756-0ef6b88557e2.png)

Reply:
Fixed in 2.0.9.